vllm/worker/model_runner.py

```diff
@@ -473,6 +487,13 @@ def _prepare_decode(
                     block_table = block_table[-sliding_window_blocks:]
                 block_tables.append(block_table)
 
+                paged_kv_indices.extend(block_table)
```

Comment:
in this case, do we even need `paged_kv_indices`? can we just pass block_table directly?

Reply:
it's a bit subtle here, `paged_kv_indices` is 1D and does not have any padding, while `block_tables` i2D and might be padded.